.github/workflows/matirx.yml

```diff
@@ -56,7 +56,7 @@ jobs:
               - type: "section"
                 text:
                   type: "mrkdwn"
-                  text: "Workflow *${{ github.workflow }}* failed on branch *${{ github.head_ref }}*\n
+                  text: "Workflow *${{ github.workflow }}* failed\n
```

Comment:
Actually we should run this job for release-* branches too.

Reply:
Reverted, but used different variable to extract branch name